src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/design/sealedshouldhavepermitslist/InputSealedShouldHavePermitsListInnerClass.java

```diff
@@ -7,12 +7,13 @@
 package com.puppycrawl.tools.checkstyle.checks.design.sealedshouldhavepermitslist;
 
 public class InputSealedShouldHavePermitsListInnerClass {
-    sealed class A {} // violation
+    sealed class A {} // violation 'Sealed classes or interfaces should explicitly declare'
```

Comment:
Please use last part of message to keep it more clear`'should explicitly declare permitted subclasses'`